packages/ui/src/components/Form/dataFormat/DataFormatEditor.tsx

```diff
@@ -125,13 +126,15 @@ export const DataFormatEditor: FunctionComponent<DataFormatEditorProps> = (props
               </DropdownList>
             </Dropdown>
             {dataFormat && (
-              <MetadataEditor
-                data-testid="dataformat-editor"
-                name={'dataformat'}
-                schema={dataFormatSchema}
-                metadata={dataFormatModel}
-                onChangeModel={(model) => handleOnChange(dataFormat.model.name, model)}
-              />
+              <div className="dataformat-metadata-editor">
+                <MetadataEditor
+                  data-testid="dataformat-editor"
+                  name={'dataformat'}
+                  schema={dataFormatSchema}
+                  metadata={dataFormatModel}
+                  onChangeModel={(model) => handleOnChange(dataFormat.model.name, model)}
+                />
+              </div>
```

Comment:
note for the future, If we find ourselves needing to do this more often, maybe we could update the `MetadataEditor` to receive a class name, so we can spare the `div` element.

Reply:
Thanks @lordrip, this was indeed causing issues - moved to enclosing component

Reply:
Oh :(